src/main/java/hanziToAnki/DeckFactory.java

```diff
@@ -53,65 +53,36 @@ private static String getOpeningHTMLTag(int tone) {
     }
 
     private static String getPinyinWithMarks(String syllable) {
```

Comment:
Can we have some tests for this?

Reply:
It's difficult and generally not recommended to test private methods. It should get tested implicitly from the test we have anyway, no?

Reply:
Okay that makes sense, could we have a comment giving an example of what it does?

Reply:
Do you think in some way it's not obvious from the method name? Could rename to something like convertSyllableFromNumbersToToneMarks?

Reply:
That is a pretty decent suggestion. Before I was just getting lost in the code, but now I see:
`int tone = Integer.parseInt("" + syllable.charAt(syllable.length() - 1));` 
and seeing that with your new method name makes me realise what's going on.

How about this:
convertNumberedSyllableToAccentedSyllable

Also, I think we're fine to test this when it's moved to "ToneHelper.java" - https://github.com/danielt998/HanziToAnki/blob/master/src/main/java/hanziToAnki/ToneHelper.java